chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/UpdateSingleListingStatusJob.java

```diff
@@ -0,0 +1,125 @@
+package gov.healthit.chpl.scheduler.job;
+
+import java.util.Date;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+import gov.healthit.chpl.auth.permission.GrantedPermission;
+import gov.healthit.chpl.auth.user.JWTAuthenticatedUser;
+import gov.healthit.chpl.domain.CertificationStatus;
+import gov.healthit.chpl.domain.CertificationStatusEvent;
+import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
+import gov.healthit.chpl.domain.ListingUpdateRequest;
+import gov.healthit.chpl.exception.ValidationException;
+import gov.healthit.chpl.manager.CertifiedProductDetailsManager;
+import gov.healthit.chpl.manager.CertifiedProductManager;
+import gov.healthit.chpl.scheduler.job.extra.JobResponse;
+
+public class UpdateSingleListingStatusJob extends QuartzJob {
+
+    // Default logger
+    private Logger logger = LogManager.getLogger("updateListingStatusJobLogger");
```

Comment:
Don't we usually make these private static final?

Reply:
The job uses "updateListingStatusJobLogger" as it's default, but since this job is called from other jobs, the user call pass in a different logger if they choose.  
```
private void setLogger(JobExecutionContext jobContext) {
    if (jobContext.getMergedJobDataMap().containsKey("logger")) {
        if (jobContext.getMergedJobDataMap().get("logger") instanceof Logger) {
            logger = (Logger) jobContext.getMergedJobDataMap().get("logger");
        }
    }
}
```